tests/validation/test_validator.py

```diff
@@ -1,47 +1,57 @@
 import warnings
+from datetime import datetime, timedelta
 
 import pandas as pd
 from toolz.functoolz import identity
 
-from fklearn.validation.validator import validator_iteration, validator, parallel_validator
+from fklearn.training.classification import lgbm_classification_learner
+from fklearn.validation import splitters, evaluators
+from fklearn.validation.validator import (
+    validator_iteration,
+    validator,
+    parallel_validator,
+)
 from fklearn.validation.perturbators import perturbator, nullify
+import pytest
 
 
 def train_fn(df):
     def p(new_df):
         return new_df.assign(prediction=0)
 
-    log = {'xgb_classification_learner': {
-        'features': ['f1'],
-        'target': 'target',
-        'prediction_column': "prediction",
-        'package': "xgboost",
-        'package_version': "3",
-        'parameters': {"a": 3},
-        'feature_importance': {"f1": 1},
-        'running_time': "3 ms",
-        'training_samples': len(df)
-    }}
+    log = {
+        "xgb_classification_learner": {
+            "features": ["f1"],
+            "target": "target",
+            "prediction_column": "prediction",
+            "package": "xgboost",
+            "package_version": "3",
+            "parameters": {"a": 3},
+            "feature_importance": {"f1": 1},
+            "running_time": "3 ms",
+            "training_samples": len(df),
+        }
+    }
 
     return p, p(df), log
 
 
 def eval_fn(test_data):
-    return {'some_score': 1.2}
+    return {"some_score": 1.2}
 
 
 def split_fn(df):
-    return [([0, 1], [[2, 3], [2], [3]]),
-            ([2, 3], [[0, 1]])], [{"fold": 1}, {"fold": 2}]
+    return [([0, 1], [[2, 3], [2], [3]]), ([2, 3], [[0, 1]])], [
+        {"fold": 1},
+        {"fold": 2},
+    ]
 
 
 perturb_fn_train = identity
-perturb_fn_test = perturbator(cols=['rows'], corruption_fn=nullify(perc=0.25))
+perturb_fn_test = perturbator(cols=["rows"], corruption_fn=nullify(perc=0.25))
 
 
-data = pd.DataFrame({
-    'rows': ['row1', 'row2', 'row3', 'row4']
-})
+data = pd.DataFrame({"rows": ["row1", "row2", "row3", "row4"]})
```

Comment:
Maybe using a fixture here?

```suggestion
@pytest.fixture
def data():
    return pd.DataFrame({"rows": ["row1", "row2", "row3", "row4"]})
```